### DISCUSSION TITLE
Activation question (chia from sources)

### DISCUSSION THREAD

**Discussion by ghost** [Chia-Network/chia-blockchain#3747] [Q&A]
What does the command `. ./activate` mean?

How to call it outside of current directory?

**Comment by mauricecyril** [comment]
The activate command starts up the virtual environment that chia runs in.

On Linux it can be invoked outside of the chia-blockchain directory with the following:

```
$ cd /home/user/chia-blockchain/ && source venv/bin/activate
```

Or
```
$ cd ~/chia-blockchain/ && source venv/bin/activate
```

**Reply by ghost** [reply]
Thanks! It works but only inside shell script (I mean if I put `source venv/bin/activate` inside bash file).

When script completes its work then (venv) and chia binary are not available anymore.

So it is not the exact equivalent of `. ./activate`.

Do you have an idea how to keep virtual environment after script execution?
